tests/src/helpers.c

```diff
@@ -207,10 +207,10 @@ int mbedtls_test_unhexify( unsigned char *obuf,
 
     while( *ibuf != 0 )
     {
-        if ( ascii2uc( *(ibuf++), &uc ) != 0 )
+        if ( mbedtls_test_ascii2uc( *(ibuf++), &uc ) != 0 )
```

Comment:
Parethesis & whitespace -> `*(ibuf++)`. 

I am not sure which is the recommended convention for this. I would leave it as is.